js/components/appcues.tsx

```diff
@@ -0,0 +1,45 @@
+import { getMetaContent } from "../util/metadata";
+import { useEffect } from "react";
+import { useLocation } from "react-router-dom";
+
+declare global {
+  const Appcues: {
+    identify(
+      userId: string,
+      properties?: Record<string, string | number | boolean>,
+    ): void;
+    page(): void;
+  };
+}
+
+// React component, put inside react router
+export const AppcuesTrackPage = (): null => {
+  const email = getMetaContent("userEmail") ?? "";
+  const appcuesUserId = getMetaContent("appcuesUserId");
+  const location = useLocation();
+  const appcuesInitialized: boolean = Object.prototype.hasOwnProperty.call(
+    globalThis,
+    "Appcues",
+  );
+  useEffect(() => {
+    if (appcuesInitialized && appcuesUserId !== null) {
+      try {
+        Appcues.identify(appcuesUserId, {
+          email,
+        });
+      } catch (error) {
+        console.warn(error);
+      }
+    }
+  }, [appcuesInitialized, email, appcuesUserId]);
+  useEffect(() => {
+    if (appcuesInitialized) {
+      try {
+        Appcues.page();
```

Comment:
Is it important that this happens after `Appcues.identify`? Probably not, but if so, I worry about race conditions and triggering these effects in the wrong order.

Reply:
I'm not sure :/ this was in response so it being called too often in glides. it's probably fine? https://github.com/mbta/glides/pull/503